Userland/Libraries/LibWeb/FileAPI/Blob.h

```diff
@@ -40,7 +40,9 @@ class Blob
     static DOM::ExceptionOr<NonnullRefPtr<Blob>> create(Optional<Vector<BlobPart>> const& blob_parts = {}, Optional<BlobPropertyBag> const& options = {});
     static DOM::ExceptionOr<NonnullRefPtr<Blob>> create_with_global_object(Bindings::WindowObject&, Optional<Vector<BlobPart>> const& blob_parts = {}, Optional<BlobPropertyBag> const& options = {});
 
+    // https://w3c.github.io/FileAPI/#dfn-size
     u64 size() const { return m_byte_buffer.size(); }
+    // https://w3c.github.io/FileAPI/#dfn-type
     String type() const& { return m_type; }
```

Comment:
Same here. What does this mean? It certainly doesn't adhere to any serenity coding style :/